tests/vmi_multus_test.go

```diff
@@ -168,12 +170,13 @@ var _ = Describe("Multus", func() {
 				Expect(err).ToNot(HaveOccurred())
 				tests.WaitUntilVMIReady(detachedVMI, tests.LoggedInCirrosExpecter)
 
-				pingVirtualMachine(detachedVMI, "10.1.1.1", "\\$ ")
+				err := testPingFromVMI(detachedVMI, testIP, "\\$ ")
+				Expect(err).ToNot(HaveOccurred(), "VMI %s failed to ping IP %s", detachedVMI.Name, testIP)
```

Comment:
All the information of the error can be included in it, avoiding the need to add formatted info here.

I mean, the error returned could be something like this:
`return fmt.Errorf("VMI %s failed to ping IP %s", detachedVMI.Name, testIP)`
And then, at this point you just need to do `Expect(err).ToNot(HaveOccurred())`